components/repository-issues.tsx

```diff
@@ -148,10 +149,23 @@ function IssueCard({
             target="_blank"
         >
             <div className="flex flex-col justify-between">
-                <div className="flex items-center justify-between">
-                    <span className="text-sm font-medium text-gray-500">
-                        {`${issue.repo}/${issue.owner}`}
-                    </span>
+                <div className="flex items-center justify-between gap-2 flex-wrap">
+                    <section className="flex gap-2 items-center">
+                        {issue.imageUrl?.length ? (
```

Comment:
since`imageUrl` isn't an array, why do we need the .length and not just check if its not undefined or null?